flink-clients/src/main/java/org/apache/flink/client/program/ContextEnvironment.java

```diff
@@ -161,6 +171,8 @@ public ContextEnvironmentFactory(Client client, List<URL> jarFilesToAttach,
 			this.userCodeClassLoader = userCodeClassLoader;
 			this.defaultParallelism = defaultParallelism;
 			this.wait = wait;
+			_plan = null;
+			_graph = null;
```

Comment:
Is there a reason for these odd names? `lastBatchPlan` and `lastStreamJobGraph` would be more appropriate. The first one is for batch programs, the latter for JobGraph created by the streaming API.